gordo_components/data_provider/ncs_reader.py

```diff
@@ -58,6 +58,7 @@ def load_series(
         to_ts: datetime,
         tag_list: List[SensorTag],
         dry_run: Optional[bool] = False,
+        remove_bad_values: Optional[bool] = True,
```

Comment:
As we have lots of 'Status codes', perhaps we should change this to `filter_status_codes = [0]` and this way we can support any future changes in status codes or other requirements without hard-coding.

Reply:
Would ```remove_status_codes = [0]``` be a better name? ```filter_status_code``` is kind of ambiguous, you have to look at the code to see if the status codes are kept or removed. @milesgranger